src/claimStatus/paymentDetail.js

```diff
@@ -559,9 +560,9 @@ function getPaymentHistoryAccordions(paymentRecords) {
           </button>
         </h3>
         <div
-          id="sect${idx}"
+          id="sect${isFuturePayment ? "Future" : "Past"}${idx}"
           role="region"
-          aria-labelledby="accordion${idx}id"
+          aria-labelledby="accordion${isFuturePayment ? "Future" : "Past"}${idx}id"
```

Comment:
[dust] - noticing `isFuturePayment ? "Future" : "Past"` is repeated a few times. low risk/low payoff but maybe could be abstracted?